samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -116,3 +116,15 @@ tests:
       - nrf54l15pdk/nrf54l15/cpuapp
     integration_platforms:
       - nrf54l15pdk/nrf54l15/cpuapp
+  sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash.pure_dts:
+    extra_args:
+      - OVERLAY_CONFIG="overlay-bt.conf"
+      - DTC_OVERLAY_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.overlay"
+      - mcuboot_CONF_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.conf"
+      - mcuboot_EXTRA_DTC_OVERLAY_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.overlay"
+    extra_configs:
+      - SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
Must be moved to `extra_args` section. Twister has problems with processing `extra_configs` if it is not started with `CONFIG_*`.
Test with command:
`${ZEPHYR_BASE}/scripts/twister -vv -T ${ZEPHYR_BASE}/samples/subsys/mgmt/mcumgr/smp_svr -s sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash.pure_dts -G`